cibuildwheel/windows.py

```diff
@@ -543,8 +542,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (remove if already exists)
             if compatible_wheel is None:
-                with suppress(FileNotFoundError):
-                    (build_options.output_dir / repaired_wheel.name).unlink()
+                (build_options.output_dir / repaired_wheel.name).unlink(missing_ok=True)
```

Comment:
```suggestion
                build_options.output_dir.joinpath( repaired_wheel.name).unlink(missing_ok=True)
```

Maybe this style is marginally nicer?